pages/lists/[listId]/contributors.tsx

```diff
@@ -69,10 +70,13 @@ const ContributorsListPage = ({ list, initialData, isError, isOwner }: Contribut
   const {
     isLoading,
     setPage,
-    setLimit,
-    setRange,
     data: { data: contributors, meta },
-  } = useContributorsList({ listId: list?.id, initialData, defaultRange: range as string });
+  } = useContributorsList({
+    listId: list?.id,
+    initialData,
+    defaultRange: range as string,
+    defaultLimit: limit as string,
```

Comment:
Looks like do a cast here we cast this number to a string to be consumed in the URL? Like brandon mentioned, I'm also curious if there's a way to prevent storing strings as numbers (especially if it'll be converted to a number eventually and cast back to a string here)

Reply:
This one is king of inevitable because it comes from the router via the querystring. It's type is `string | string[] | undefined`, so in this case, this would need to become most likely this

`defaultLimit: Number(limit as string),`